src/mscorlib/src/System/Reflection/CustomAttribute.cs

```diff
@@ -2231,7 +2231,7 @@ internal static Attribute[] GetCustomAttributes(RuntimeAssembly assembly, Runtim
 
             bool all = caType == (RuntimeType)typeof(object) || caType == (RuntimeType)typeof(Attribute);
 
-            if (!all && s_pca.GetValueOrDefault(caType) == null && !IsSecurityAttribute(caType))
+            if (!all && s_pca.ContainsKey(caType) == false && !IsSecurityAttribute(caType))
```

Comment:
can we do `!s_pca.ContainsKey(caType)` instead of `s_pca.ContainsKey(caType) == false` please?